e2e/test/scenarios/sharing/sharing-reproductions.cy.spec.js

```diff
@@ -630,15 +630,15 @@ describeEE("issue 26988", () => {
     });
 
     openStaticEmbeddingModal({
-      activeTab: "appearance",
+      activeTab: "lookAndFeel",
       previewMode: "preview",
       acceptTerms: false,
     });
 
     cy.wait("@previewDashboard");
     getIframeBody().should("have.css", "font-family", "Lato, sans-serif");
 
-    cy.findByLabelText("Playing with appearance options")
+    cy.findByTestId("look-and-feel-settings")
```

Comment:
The reason this is changed is because the new design removes the header entirely (there used to be a header in the red circle), so I need to get it by something else. I think `testid` is appropriate in this situation.
![Screenshot 2024-06-17 at 9 41 07 PM](https://github.com/metabase/metabase/assets/1937582/ab03c8ca-ddb9-4e4a-b00d-f1b9cf25cc68)